src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/MarkTaskReconfigurationService.java

```diff
@@ -88,7 +89,7 @@ public TaskOperationResponse performOperation(TaskOperationRequest taskOperation
 
     private boolean isReconfigurable(String caseId) {
         List<ConfigurationDmnEvaluationResponse> results = caseConfigurationProviderService
-            .evaluateConfigurationDmn(caseId, null);
+            .evaluateConfigurationDmn(caseId, emptyMap());
```

Comment:
maybe a unit test needs updating to cover this?  I.e. it should fail if it goes back to null.

Reply:
Hi Ravi... I added unit test to check this behaviour using captors. Could you please verify and let me know if thats okay.

Reply:
looks good.